### ISSUE TITLE
HOOBs Still running on Mac

### ISSUE THREAD

**Issue by pcdocdwb** [hoobs-org/HOOBS#836]
I was previously running HOOBS Version 3.2.6 Custom Install on my imac, but because I am always on a VPN, it didn't connect all the time.  So I purchased a new Hoobs in a Box and have it connected.  I thought I cleared out al my hoobs settings on my Mac and even created a new account, but when I log into Hoobs:local, it is still logging into my iMac.  How do I permanently disable Hoobs on my Mac so I can see and set up the Hoobs in a Box?  Thank you.  

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```
10/9/2020, 2:30:27 PM HOOBS listening on port 80.
10/9/2020, 2:30:49 PM Loaded plugin "homebridge-ring".
10/9/2020, 2:30:49 PM Bridge is running on port 51826.
10/9/2020, 2:30:49 PM [Ring] Request to https://app.ring.com/rhq/v1/devices/v1/locations failed:
10/9/2020, 2:30:49 PM [Ring] Error: Refresh token is not valid.  Unable to authenticate with Ring servers.  See https://github.com/dgreif/ring/wiki/Refresh-Tokens
    at RingRestClient.getGrantData (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:120:15)
    at RingRestClient.<anonymous> (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:124:36)
    at Generator.next (<anonymous>)
    at /home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:23:12)
    at RingRestClient.getAuth (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:123:16)
    at new RingRestClient (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:100:33)
    at new RingApi (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/api.js:26:27)
    at RingPlatform.<anonymous> (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/homebridge/ring-platform.js:143:53)
10/9/2020, 2:30:49 PM [Ring] Error connecting to API
10/9/2020, 2:30:49 PM [Ring] Error: Refresh token is not valid.  Unable to authenticate with Ring servers.  See https://github.com/dgreif/ring/wiki/Refresh-Tokens
    at RingRestClient.getGrantData (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:120:15)
    at RingRestClient.<anonymous> (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:124:36)
    at Generator.next (<anonymous>)
    at /home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:23:12)
    at RingRestClient.getAuth (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:123:16)
    at new RingRestClient (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:100:33)
    at new RingApi (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/api.js:26:27)
    at RingPlatform.<anonymous> (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/homebridge/ring-platform.js:143:53)
10/9/2020, 2:30:49 PM (node:602) UnhandledPromiseRejectionWarning: Error: Refresh token is not valid.  Unable to authenticate with Ring servers.  See https://github.com/dgreif/ring/wiki/Refresh-Tokens
10/9/2020, 2:30:49 PM at RingRestClient.getGrantData (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:120:15)
10/9/2020, 2:30:49 PM at RingRestClient.<anonymous> (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:124:36)
10/9/2020, 2:30:49 PM at Generator.next (<anonymous>)
10/9/2020, 2:30:49 PM at /home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:27:71
10/9/2020, 2:30:49 PM at new Promise (<anonymous>)
10/9/2020, 2:30:49 PM at __awaiter (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:23:12)
10/9/2020, 2:30:49 PM at RingRestClient.getAuth (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:123:16)
10/9/2020, 2:30:49 PM at new RingRestClient (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/rest-client.js:100:33)
10/9/2020, 2:30:49 PM at new RingApi (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/api/api.js:26:27)
10/9/2020, 2:30:49 PM at RingPlatform.<anonymous> (/home/hoobs/.hoobs/node_modules/homebridge-ring/lib/homebridge/ring-platform.js:143:53)
10/9/2020, 2:30:49 PM (node:602) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
10/9/2020, 2:30:49 PM (node:602) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
10/9/2020, 2:30:49 PM (node:602) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
10/9/2020, 2:43:01 PM Got SIGINT, shutting down Bridge...
10/9/2020, 2:43:07 PM Loaded plugin "homebridge-ring".
10/9/2020, 2:43:07 PM Bridge is running on port 51826.
10/9/2020, 2:43:08 PM [Ring] 404 from endpoint https://api.ring.com/clients_api/ring_devices
10/9/2020, 2:43:08 PM [Ring] Session hardware_id not found.  Creating a new session and trying again.
10/9/2020, 2:43:10 PM [Ring] Creating location socket.io connection
10/9/2020, 2:43:10 PM [Ring] Ring connected to socket.io server
10/9/2020, 2:43:10 PM [Ring] Configuring 16 cameras and 39 devices for location "C&D Home" - locationId: be25cd86-fdc9-4d49-80cf-8c4694aa4d6b
10/9/2020, 2:43:10 PM [Ring] Hidden accessory 007e18f2-c6a2-4e21-87ba-4e0e88f10f62 adapter.zigbee ZigBee onboard adapter
10/9/2020, 2:43:10 PM [Ring] Hidden accessory 2e00c149-dd97-4f5a-b84c-7fb77ccf5a7a access-code.vault Code Vault
10/9/2020, 2:43:10 PM [Ring] Adding new accessory 1d5b9dfb-376a-4045-9898-b285504a3184 hub.redsky Base Station
10/9/2020, 2:43:10 PM [Ring] Adding new accessory f713b1a5-10a3-4074-a48d-4dfa8c31a44e security-panel Ring Alarm
10/9/2020, 2:43:10 PM [Ring] Hidden accessory d8b0ba17-0fc1-4085-9d47-677b57b03787 access-code Security Access Code
10/9/2020, 2:43:10 PM [Ring] Hidden accessory adccdc2a-c32f-4e26-8a5e-326b8eec528b adapter.zwave Z-Wave Adapter
10/9/2020, 2:43:10 PM [Ring] Adding new accessory 685cafeb-afd6-4745-969d-e5505f0b75dc security-keypad Entryway Keypad
10/9/2020, 2:43:10 PM [Ring] Adding new accessory 6229a495-7de7-4f6e-ac6d-8a22803a2a12 sensor.contact Back Door
10/9/2020, 2:43:10 PM [Ring] Hidden accessory 279649de-e2b6-473e-a1a7-d915bfe8e3c1 range-extender.zwave LR RE
10/9/2020, 2:43:10 PM [Ring] Adding new accessory 1f4f23b1-2729-4db3-9e74-ff32acf30431 sensor.motion Office
10/9/2020, 2:43:10 PM [Ring] Adding new accessory 7ba1dd62-307b-41b4-94b1-fc108ab887df sensor.contact Front Door
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 2844d017-4260-467f-b114-a898b9476c1d sensor.contact Side Exterior Door
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 592c70a9-6442-4935-a77b-247af85c1c26 sensor.contact Slider
10/9/2020, 2:43:11 PM [Ring] Adding new accessory baf42ce2-6cae-4ac9-89d8-a7c095283bdc sensor.motion Bedroom
10/9/2020, 2:43:11 PM [Ring] Adding new accessory c4a71bf2-f5ae-4614-9418-ad0099d88a87 sensor.motion Spare  Bedroom
10/9/2020, 2:43:11 PM [Ring] Adding new accessory a3767c56-2abe-40b5-b08a-f19332af72a2 sensor.motion Living Room
10/9/2020, 2:43:11 PM [Ring] Adding new accessory dd160c07-59d8-4b01-80ed-3cc59d7e5cb2 sensor.motion Sun Room
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 563f9116-1014-4270-bdd8-b5d9f288ba34 sensor.contact Front Window
10/9/2020, 2:43:11 PM [Ring] Adding new accessory e4a4ca33-3151-422d-8441-8759b4bed7b6 sensor.contact Side Interior Door
10/9/2020, 2:43:11 PM [Ring] Hidden accessory 48c0458b-c549-4f2b-8a7a-0ccdd7c30202 range-extender.zwave Kitchen RE
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 90540e83-1df7-4336-8a0e-343960252506 sensor.contact Side Window
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 13384dde-d902-4ff5-9f5e-8b9af19f47b0 sensor.contact Back Window
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 3408fa26-f1bb-415e-8560-103914eef977 doorbell_scallop Side Door Exterior Bell
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 4c527976-8284-4e30-aea9-07e2138d9c9a doorbell_scallop Front Door Bell
10/9/2020, 2:43:11 PM [Ring] Adding new accessory b37789a8-b753-4166-b855-bb9d75fc08fb lpd_v2 Side Interior Bell
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 015ba3b7-e6ba-4f47-b255-9da7616a9ebf floodlight_v2 Backyard North Cam
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 9ce9651f-f17f-4e03-9020-6a0b896f2ad5 floodlight_v2 Backyard South Cam
10/9/2020, 2:43:11 PM [Ring] Adding new accessory e967fb54-0d11-4be0-a414-178cb46a921f stickup_cam_v4 Shed Ext Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 93b204fb-8c63-4efc-9c9c-4b462d225c9c stickup_cam_v4 Side Ext Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 9e5726c5-f169-4084-b509-3d07a796292a stickup_cam_v4 Front Ext Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 07ab0828-ee82-4969-8316-eb19d4adf36e cocoa_camera MBR Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 3ed3578c-3192-44a3-9d6d-82322643a3d9 stickup_cam_lunar Kitchen Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 6fcbe997-7a4f-425b-8ff2-57295208fff7 stickup_cam_lunar DR Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 54160116-7633-4be4-be15-070a0fbef184 stickup_cam_lunar LR Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory c9ecbc40-4ede-4582-9a4b-7687dd548110 stickup_cam_lunar Office
10/9/2020, 2:43:11 PM [Ring] Adding new accessory f1d47b9a-500e-4a3b-ae3e-ef8b2671110e stickup_cam_v4 Deck Ext Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory bb8688a6-698a-4e01-bab3-8e7ecdbbcf11 stickup_cam_v4 Backyard Ext Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory c1b02c5b-c3ed-4479-9ae8-52aa218ad4f0 stickup_cam_lunar Sun Room Camera
10/9/2020, 2:43:11 PM [Ring] Adding new accessory 1bcf767c-38df-4c60-9a93-48c34f6b4d2d chime_pro Upstairs
10/9/2020, 2:52:02 PM [Ring] Sun Room Camera Detected Motion
10/9/2020, 2:52:14 PM [Ring] LR Camera Detected Motion
10/9/2020, 2:52:16 PM [Ring] DR Camera Detected Motion
10/9/2020, 2:52:41 PM [Ring] Side Interior Bell Detected Motion. Loading snapshot before sending event to HomeKit
10/9/2020, 2:52:45 PM [Ring] Kitchen Camera Detected Motion
10/9/2020, 2:52:45 PM [Ring] Office Detected Motion
10/9/2020, 2:52:49 PM [Ring] Side Door Exterior Bell Detected Motion
10/9/2020, 2:54:11 PM [Ring] DR Camera Detected Motion
10/9/2020, 2:54:16 PM [Ring] Sun Room Camera Detected Motion
10/9/2020, 2:55:15 PM [Ring] Kitchen Camera Detected Motion
```

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
{
    "server": {
        "port": 80,
        "origin": "*",
        "autostart": 5,
        "home_setup_id": "X-HM://0023ISYWYCDH2",
        "polling_seconds": 5
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hazed-infused",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": "06451",
        "latitude": "41.537590",
        "longitude": "-72.801102"
    },
    "bridge": {
        "name": "HOOBS",
        "port": 51826,
        "pin": "031-45-154",
        "username": "66:AD:A4:37:A0:8B"
    },
    "description": "C&D Apple Homekit",
    "ports": {},
    "accessories": [],
    "platforms": [
        {
            "platform": "Ring",
            "plugin_map": {
                "plugin_name": "homebridge-ring"
            },
            "refreshToken": 
        }
    ]
}
```

**Additional context**

**Comment by askovi** [comment]
Open a Terminal on your mac and run this command to uninstall HOOBS:

`sudo npm uninstall -g @hoobs/hoobs`

Additionally you can check the ip of the HOOBS Box on your router's admin page and connect to it directly like this: http://192.168.x.x

**Comment by pcdocdwb** [comment]
Thanks.  I ran that command successfully.  I also connected directly to the Hoobs box via the IP.  It all looks good.  However, if my Mac is still connected to the VPN, I still can't stream the cameras in Homekit.  As soon as I turn off the VPN on my Mac, the cameras stream fine in Homekit.  Hoobs is now definitely running on the hoobs box, and I am definitely logged into Hoobs on the Hoobs box.  How can the VPN on the Mac still be affecting the streaming when Hoobs is on the Hoobs box?
